<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QualityEmployeeWindow</class>
 <widget class="QMainWindow" name="QualityEmployeeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Quality Employee</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="ComplainsDisplay" name="complainsDisplay" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>780</width>
      <height>360</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>781</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Notes</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="notes_textEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QComboBox" name="state_comboBox">
         <item>
          <property name="text">
           <string>Decline</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Completed</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="setState_pushButton">
         <property name="text">
          <string>Set state</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ComplainsDisplay</class>
   <extends>QWidget</extends>
   <header>complainsdisplay.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
